base_layer/core/src/chain_storage/blockchain_database.rs

```diff
@@ -66,7 +66,7 @@ use std::{
 };
 use tari_common_types::{chain_metadata::ChainMetadata, types::BlockHash};
 use tari_crypto::tari_utilities::{hex::Hex, ByteArray, Hashable};
-use tari_mmr::{MerkleMountainRange, MutableMmr};
+use tari_mmr::{pruned_hashset::PrunedHashSet, MerkleMountainRange, MutableMmr};
 use uint::static_assertions::_core::ops::RangeBounds;
```

Comment:
May be fix this while we're at it

```suggestion
use std::ops::RangeBounds;
```